script.js

```diff
@@ -0,0 +1,36 @@
+const firstName = document.getElementById("firstName");
+const lastName = document.getElementById("lastName");
+const age = document.getElementById("age");
+const email = document.getElementById("email");
+const phoneNum = document.getElementById("phoneNum");
+const male = document.getElementById("male");
+const female = document.getElementById("female");
+let errorMsg = document.getElementsByTagName("p")[0];
+let submit = document.getElementById("submit");
+let form = document.getElementById("form");
+
+const validateInput = (input) => {
+  if (input === "age" && age.value < 18 && age.value < 100) {
+    errorMsg.innerText = "Age should be greater than 18";
+  } else if (input === "phoneNum" && phoneNum.value.length != 10) {
+    errorMsg.innerText = "Phone number should be of 10 digits";
+  } else {
+    errorMsg.innerText = "";
+  }
+};
+
+const formHandler = (event) => {
+  submit.addEventListener("click", () => {
+    submit.innerText = "Clicked";
```

Comment:
@Arzoo-25 Can we check if `errorMsg.innerText ` is empty string  and then change the button text in place of click `EventListener`?